src/scripts/main.js

```diff
@@ -1,3 +1,45 @@
 'use strict';
 
-// write your code here
+const tableData = createDataArr('tr');
+const table = () => {
+  const newTable = [];
+
+  for (let i = 0; i < tableData.length; i++) {
+    (i === 0 || i === tableData.length - 1)
+      ? newTable.push(getRowTexts('th', i))
+      : newTable.push(getRowTexts('td', i));
```

Comment:
it's very cool that you know how to write like this, but the code should still be as readable as possible for others, so that every member of the team can maintain your code in the future and spend the least amount of time understanding it

`i === 0` move to variable
`i === tableData.length - 1` move to variable

how about if else?

Reply:
I thought about moving them to vars but then decided that it'll be overthinking)
And I used `if` but then decided to use ternary)
Seems like I still can't choose the most readable and obvious way to implement)
Thanks for paying attention and for your advice!